lib/Analysis/Alias.cpp

```diff
@@ -34,8 +34,9 @@ LogicalResult SharedMemoryAliasAnalysis::visitOperation(
       return success();
   }
 
-  // Only LocalAllocOp creates a new buffer.
-  if (isa<triton::gpu::LocalAllocOp>(op)) {
+  // New buffer creation ops.
+  if (isa<triton::gpu::LocalAllocOp, triton::ExperimentalTensormapAllocOp>(
```

Comment:
I find it a bit weird to have to create a new allocation op just for TMA. Can we lower to the existing `LocalAllocOp`?